### PR TITLE
TB-58 Fix hapi-fhir config importer

### PR DESCRIPTION
**Pull request by nour-borgi**

### CODE REVIEW

File: swarm.sh [hunk 1]
```diff
@@ -48,7 +48,12 @@ main() {
     config::await_service_removed instant_message-bus-kafka-config-importer
     config::await_service_removed instant_sante-mpi-config-importer
     config::await_service_removed instant_hapi-fhir-config-importer
-     
+
+    log info "Restarting HAPI FHIR.."
+    try "docker service scale instant_hapi-fhir=0" "Error scaling down hapi-fhir to update the IG"
+    sleep 10
```
Comment (Mark-Labuschagne) [inline swarm.sh:54]: As far as I know, scaling a service down does a convergence check. Do you get errors if not using this sleep?

Comment (nour-borgi) [reply swarm.sh:54]: let me test it, and it works I will push again 👍🏼

Comment (nour-borgi) [reply swarm.sh]: It works! nice Thanks Mark, I updated that 🚀

**Review by Mark-Labuschagne — APPROVED** [review]

**Review by michaelloosen — APPROVED** [review]
LGTM

**Pull request merged**